pkg/repository/clone_mirror.go

```diff
@@ -306,27 +306,6 @@ func cloneComponents(repo *V1Repository,
 		compManifests[name] = newManifest
 	}
 
-	// Download TiUP binary
-	for _, goos := range options.OSs {
```

Comment:
Why delete this part?

Reply:
It's of no use in V1. This part doesn't change add `tiup` to manifests. But with V1 repository, tiup update self only read from manifests.